airflow/utils/dag_processing.py

```diff
@@ -651,6 +659,8 @@ def __init__(self,
 
         self._log = logging.getLogger('airflow.processor_manager')
 
+        self.waitables = {self._signal_conn: self._signal_conn}
```

Comment:
Can this be converted to a set?

Reply:
No - cos

``
self.waitables[processor.waitable_handle] = processor
```

Reply:
Just feels weird as { k: v} where k=v ! Haven't looked at the entire PR so it might make sense after it.